torchci/pages/[repoOwner]/[repoName]/commit/[sha].tsx

```diff
@@ -67,6 +68,10 @@ export default function Page() {
       ? "ExecuTorch"
       : `${repoOwner}/${repoName}`;
 
+  useEffect(() => {
```

Comment:
This one doesn't seem to update the title? Or am I looking in the wrong place?

Reply:
It does, on line 72